otx/algorithms/segmentation/adapters/mmseg/utils/builder.py

```diff
@@ -58,9 +59,12 @@ def build_segmentor(
     model = origin_build_segmentor(model_cfg, train_cfg=train_cfg, test_cfg=test_cfg)
     model = model.to(device)
 
-    checkpoint = checkpoint if checkpoint else config.pop("load_from", None)
+    checkpoint = checkpoint if checkpoint else config.get("load_from", None)
+    config.load_from = checkpoint
```

Comment:
I'm not sure this build_model() is the right place to set config.load_from, but let's align the policy along with other tasks in the near future.

Reply:
Thanks. I will think about it where to load.